js/ProgressView.windows.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ * @flow strict-local
+ */
+'use strict';
+
+import {requireNativeComponent, StyleSheet} from 'react-native';
+import React from 'react';
+import type {ImageSource} from 'react-native/Libraries/Image/ImageSource';
+import type {ColorValue} from 'react-native/Libraries/StyleSheet/StyleSheetTypes';
+import type {ViewProps} from 'react-native/Libraries/Components/View/ViewPropTypes';
+
+const RNCProgressView = requireNativeComponent(
+  'RNCProgressViewWindows',
+);
+
+type WindowsNativeProps = $ReadOnly<{|
+  ...ViewProps,
+  progressViewStyle?: ?('default' | 'bar'),
+  progress?: ?number,
```

Comment:
Why the question mark right before number? Isn't the one before the colon enough to denote an optional argument?